test/NavDropdownSpec.js

```diff
@@ -59,4 +59,15 @@ describe('<NavDropdown>', () => {
 
     wrapper.assertSingle('a#test-id');
   });
+
+  it('should support as as prop', () => {
+    const wrapper = mount(
+      <NavDropdown as="li" id="test-id" title="title">
+        <DropdownItem eventKey="1">Item 1</DropdownItem>
+      </NavDropdown>,
+    );
+
+    wrapper.assertSingle('li');
```

Comment:
```suggestion
```

The second assert takes care of the first assert.

Reply:
Didn't notice that haha thanks!